tests/Unit/Config/XmlConfigFileLoaderTest.php

```diff
@@ -87,7 +93,7 @@ public function test_it_works_if_no_dynamics_are_present()
         $loader = $this->createConfigLoader();
 
         $config = $loader->load(self::TEMP_FILE, []);
-        $this->assertEquals([ 'FOO' => 'bar'], $config->getConstants());
+        $this->assertEquals(['FOO' => 'bar'], $config->getConstants());
```

Comment:
```suggestion
        self::assertEquals(['FOO' => 'bar'], $config->getConstants());
```

Please fix this in the whole project.

Reply:
Already done, is part of a later pr